cmd/gocritic/testdata/unnamedResult/negative_tests.go

```diff
@@ -0,0 +1,29 @@
+package checker_test
+
```

Comment:
Negative tests also need a few declarations with named results with 2 and more results that would emit warning
if there were unnamed.